lxd/daemon.go

```diff
@@ -826,6 +826,12 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 			resp = response.NotFound(fmt.Errorf("Method %q not found", r.Method))
 		}
 
+		// Log a warning if endpoint is part of the main API, and therefore should be counted fot the API metrics.
```

Comment:
typo "fot"

Reply:
do we even need this logging anymore now we force the metrics type on the API handler?

Reply:
I was asking myself the same thing. It would be nice if we could check the callback function was called somehow, but I don't see how we would do this.

Reply:
OK think we should just remove this part for now.

Reply:
Done!